packages/astro/src/core/sync/index.ts

```diff
@@ -65,7 +65,7 @@ export async function sync(
 	const tempViteServer = await createServer(
 		await createVite(
 			{
-				server: { middlewareMode: true, hmr: false },
+				server: { middlewareMode: true, hmr: false, watch: { ignored: ['**/*'] } },
```

Comment:
As explained in the PR description, this isn't needed, but I updated anyways so it's consistent with the `createViteLoader` code.